tests/test_lsthing.py

```diff
@@ -399,6 +399,7 @@ def test_005_create_interactions(self):
         fresh_proj_1 = Project.get_by_code(proj_1.code_name, self.client, Project.ls_type, Project.ls_kind)
         assert len(fresh_proj_1.links) == 1
         assert len(fresh_proj_1._ls_thing.second_ls_things) == 1
+        assert fresh_proj_1._ls_thing.second_ls_things[0].ls_kind == f'{Project.ls_type}_{Project.ls_type}'
```

Comment:
Add a test to verify that the ls_type and ls_kind of the objects being passed in are still used as the default